commit 8f7a0ba3373a983720cc15f50c839e544b99b231
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2019-04-09T12:18:16-07:00 (6 months ago)
    
    TRITON-1381 mockcloud server_reboot using the wrong assert
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/lib/backends/dummy/tasks/server_reboot.js b/lib/backends/dummy/tasks/server_reboot.js
index 5337305..6553326 100644
--- a/lib/backends/dummy/tasks/server_reboot.js
+++ b/lib/backends/dummy/tasks/server_reboot.js
@@ -10,7 +10,7 @@
 
 var spawn = require('child_process').spawn;
 
-var assert = require('assert');
+var assert = require('assert-plus');
 
 var common = require('../common');
 var shared = require('./shared');
diff --git a/package.json b/package.json
index 910aca3..0332b44 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cn-agent",
   "description": "Triton Compute Node Agent",
-  "version": "2.12.0",
+  "version": "2.12.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
